### PR TITLE
Added a list of games banned in Saudi Arabia

### PR DESCRIPTION
**Pull request by BratPfanneTV**

### CODE REVIEW

File: data/games/bannedGames/saudi_arabia/bannedList.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
+{
+  "description":"A list of video games banned in Saudi Arabia",
+  "games":[
+    {
+    "name":"God of War (franchise)",
+    "reason":"Banned for a few weeks due to the presence of interactive sex scenes, partial nudity and sexual themes, as well as the word "God" in the title. In 2018, the newest game in the series was approved for release."
```
Comment (hugovk) [inline data/games/bannedGames/saudi_arabia/bannedList.json]: Fix a quote error:

```suggestion
    "reason":'Banned for a few weeks due to the presence of interactive sex scenes, partial nudity and sexual themes, as well as the word "God" in the title. In 2018, the newest game in the series was approved for release.'
```

https://travis-ci.org/dariusk/corpora/builds/660543011

Comment (hugovk) [reply data/games/bannedGames/saudi_arabia/bannedList.json]: Sorry, that also didn't work: https://travis-ci.org/dariusk/corpora/builds/660625599

Maybe try it like: `"reason":"......'...'......"`

Or escape the double quotes:  `"reason":"......\"...\"......"`

**Comment by BratPfanneTV** [comment]
I guess now it should work?

**Comment by hugovk** [comment]
✅ Yes, now passing! https://travis-ci.org/dariusk/corpora/builds/660630969

**Pull request merged**